activitysim/abm/models/vehicle_type_choice.py

```diff
@@ -380,6 +391,12 @@ def iterate_vehicle_type_choice(
             len(choosers),
         )
 
+        # filter columns of alts and choosers
+        if len(model_settings.get("COLS_TO_INCLUDE_IN_CHOOSER_TABLE")) > 0:
```

Comment:
Need to set default for get function to be array.  i.e. `model_settings.get("COLS_TO_INCLUDE_IN_CHOOSER_TABLE", [])`.  Here and below.